ui/src/views/Provisioners/ViewWorkers/index.jsx

```diff
@@ -36,6 +43,10 @@ import workersQuery from './workers.graphql';
         parse(location.search.slice(1)).filterBy === 'quarantined'
           ? true
           : null,
+      workerState:
+        parse(location.search.slice(1)).filterBy in STATES
```

Comment:
to be honest, I haven't seen this python approach being popular in js to check if key exists. As `in` operator also checks props down the prototypes inherited, like:

```
const a = {"one": 1}
"toString" in a // returns true here, because a.toString() is defined in object's prototype
```

It would be safer to use `.hasOwnProperty()` or just simplify it to do a check against allowed values like `Object.values(STATES).includes(location.search..)` 

wdyt?